src/main/java/de/adorsys/smartanalytics/api/WrappedBooking.java

```diff
@@ -52,7 +52,28 @@
     private Cycle cycle;
     // *****************************************
 
+    // @Builder.Default does not work. Shall be the clean solution.
     private Set<String> usedRules = new HashSet<>();
+    
+    /**
+     * We need to override the builder to perform any
+     * constructor operations we need. Since we know builder uses
+     * the all args constructor and uses null to initialized all 
+     * unset fields.
+     * 
+     * @return
```

Comment:
please either document the return value or remove the javadoc. Otherwise, it wouldn't pass checks by releasing to Nexus.
P.S. Lombok builders have many problems, in general their usage is not recommended.

Reply:
@tadschik makes intensive use of lombok. FOr this case is the fix solving the problem?

Reply:
We all use Lombok. The only exception from its usage I make: I do not recommend to use its builders. I've found following problems with builders in Lombok:
1. They are not using field default values. Builder.Default are not working properly.
2. It works strange with collections. Sometimes they are initialized with empty list, sometimes not.
3. It doesn't work at all if you have final fields. Compilation passes, but in runtime, you got an error: it initializes everything with null first and then tries to set value.
4. It doesn't work for objects that are descendants of other objects: you cannot use it to initialize parent fields. You cannot use it if parent object has no default constructor either.
5. It doesn't work if you would like to make immutable class w/o getters. Don't know why.
So this feature of Lombok are quite risky, especially in open apis.

Reply:
P.s. Javadoc still to be fixed :)